packages/paste-core/components/checkbox/src/CheckboxDisclaimer.tsx

```diff
@@ -10,22 +10,24 @@ export interface CheckboxDisclaimerProps
   errorText?: string | React.ReactNode;
 }
 
-const CheckboxDisclaimer: React.FC<CheckboxDisclaimerProps> = ({children, errorText, ...props}) => {
-  return (
-    <>
-      <Box backgroundColor="colorBackground" borderRadius="borderRadius20" padding="space50">
-        <Checkbox {...props} hasError={errorText != null}>
-          {children}
-        </Checkbox>
-      </Box>
-      {errorText && (
-        <Box marginTop="space30" marginLeft="space50">
-          <HelpText variant="error">{errorText}</HelpText>
+const CheckboxDisclaimer = React.forwardRef<HTMLDivElement, CheckboxDisclaimerProps>(
+  ({children, errorText, ...props}, ref) => {
+    return (
+      <>
+        <Box backgroundColor="colorBackground" borderRadius="borderRadius20" padding="space50" ref={ref}>
```

Comment:
Unsure myself here, asking for other opinions: should we pass ref directly to the Checkbox in this case?

Reply:
That actually makes more sense to me. The rest of the the disclaimer, i.e. Box is just a visual element.